Roddy.iml

```diff
@@ -1,81 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Actually, I did not notice and I am very surprised, because apparently IntelliJ does not need this. The file is gone, but the project module -- now simply imported from the Gradle model -- is present. Not sure yet, what that might mean.

Reply:
I am more or less the only developer left and the system needed to be changed, because IntelliJ apparently does not support the old way with one module including sources and tests. All this is now automatically modelled as submodules and there is no way to change this, because IntelliJ automatically reimports from the Gradle-file.

I suggest we leave this like it is here in the PR.